card_lookup.py

```diff
@@ -24,11 +24,19 @@ def clean_disc_string(string):
 def process_card_lookup(matches):
     with DBConnect(settings.CARD_DB) as conn:
         cur = conn.cursor()
-        sql = 'SELECT * FROM cards WHERE card_name = ? COLLATE NOCASE'
+        # sql = 'SELECT * FROM cards WHERE card_name = ? COLLATE NOCASE'
+        sql = "SELECT * FROM cards WHERE card_name LIKE '%?%'"
+
         results = []
         for match in matches:
             for group in match:  # a 2-tuple, [[group 1]] and \[\[group2\]\]
-                row = cur.execute(sql, [group]).fetchone()
+                group_words = group.translate(str.maketrans(","," ")).split()
```

Comment:
```suggestion
                if not group:
                  continue
                group_words = group.translate(str.maketrans(","," ")).split()
                group_words = ['%' + w + '%' for w in group_words]
```
Executing the sql statement throws an error when group is empty (and in most cases, 1 of them almost always is) because the statement expects at least 1 value to replace the ? placeholder (when group is empty string, split() returns empty list to group_words, so no values are passed later to the sql statement) The `if ...` statement is for this case.

Modify the values in group_words here to be surrounded by %